Tests/Functional/DebugModeTest.php

```diff
@@ -0,0 +1,690 @@
+<?php
```

Comment:
tyjo, fakt by se mi líbilo, kdyby se tyhle testy roztrhly na 2 části

1) otestovat nějakou classu, že dokáže vzít datadir, zabalit ho a spolu s nějakejma tagama ho nahrát do file uploadů
2) otestovat runner/executor, že zavolá ^^^ classu ve správnou chvíli se správnejma parametrama (nebude nic reálně uploadovat ani stahovat z file uploadů)

tím by se to celý zkrátilo a zjednodušilo?

Reply:
no pridat samostany testy na storeZip urcite muzu, ale tohle je funkcionalni test, kterej opravdu nechci rusit - nejde jen o o to, jestli to zavola spravnou classu se spravnym parametrama, ale kdy ji zavola, kolikrat, jestli ji spravne preda filtr

Reply:
to se určitě dá otestovat i bez toho, aby se cokoliv reálně uploadovalo (stejně jako se dělaj storageApiService mocky)

Reply:
spíš mi ale šlo o to vypíchnou, jestli by to nešlo trochu jinak přeskládat, že by to mohlo bejt jednodušejc uchopitelný